src/main.cpp

```diff
@@ -35,6 +39,12 @@ int main(void)
     }
 #endif
 
+#ifdef CONFIG_AOS_REBOOT
```

Comment:
Let's have it always for not native applications.

Reply:
Done